src/Sarif.Driver/OrderedFileSpecifier.cs

```diff
@@ -47,6 +47,7 @@ private IEnumerable<IEnumeratedArtifact> EnumeratedArtifacts()
         {
             string normalizedSpecifier = Environment.ExpandEnvironmentVariables(this.specifier);
 
+            // normalizedSpecifier might be changed if it contains URL Percent-Encoding characters.
```

Comment:
Please try to remember to use single quotes to indicate named things.

i.e., normalizedSpecifier is not an English word, it is the name of a variable. So for readability say. Note I fixed a casing problem as well:

// 'normalizedSpecifier' might be changed if it contains URL percent-encoded characters.

I repeat this code review note a lot. If you do not understand it, let's please talk about this language subtlety at our next stand-up.

@Stacy Wray, FOCR

Reply:
Update the comment